Dangerfile

```diff
@@ -1,25 +1,26 @@
 # Sometimes its a README fix, or something like that - which isn't relevant for
 # including in a CHANGELOG for example
-declared_trivial = pr_title.include? "#trivial"
+
+has_app_changes = !files_modified.grep(/lib/).empty?
+has_test_changes = !files_modified.grep(/spec/).empty?
 
 if ["KrauseFx", "orta"].include?(pr_author) == false
-  warn("Author @#{pr_author} is not a contributor")
+  warn "Author @#{pr_author} is not a contributor"
 end
 
 if (pr_body + pr_title).include?("WIP")
-  warn("Pull Request is Work in Progress")
+  warn "Pull Request is Work in Progress"
```

Comment:
I think those are cases for using them full stop, but that not using an awesome language feature because one day you might add an extra parenthese is a shame. I could imagine this code  using them, `declared_trivial = pr_title.include? "#trivial" || !has_app_changes` but I don't think that becomes particularly confusing to read because of the lack of `(` & `)`